### DISCUSSION TITLE
Submission Issue on EvalAI Platform

### DISCUSSION THREAD

**Discussion by TomatoPicker278** [empriselab/RCareWorld#175] [Other Q&A] [answered]
Hello!

On November 19th at 11:09:20 AM (London time), I submitted a ZIP file to the EvalAI platform. Could you please help me check if my submission can run successfully?

Currently, I am unable to test my program locally in Docker. This is because my program requires the use of a camera and the `GetRGB()` function. However, I have discovered that if I do not append `-g` to the command `python <my_test_bathing.py>` when running it, the `GetRGB()` function of the camera fails to work, resulting in a blank capture.

As such, I am unsure if the Docker image I built will execute my Python program correctly. I kindly request the organizers to help test it to ensure it runs as expected.

**Comment by shxingch** [comment, marked as answer]
Hi @TomatoPicker278 ! We have received your submission file and run it. 

For the competition submission format, we want participants to write all algorithms into a Docker container, ensuring it can be correctly executed through bash `run_python.shell `with JSON output. Then save this Docker container, generate a compressed file, use our script for secondary compression, and upload it.
We checked your files and found multiple folders and a ZIP file. We're not sure if it's an upload issue, but we discovered two Docker files in this file. We ran them manually, and it seems that `unpacked_docker `folder is the Docker file you want to submit. We ran this Docker container but encountered an error. The error is `"IndexError: list index out of range".` If you want to see detailed error content, please provide your email or email us and we will send you the specific logs.
Currently, the test scores have been submitted to the leaderboard. Please contact us if you have any questions!
